src/components/reusable/notification/notification.ts

```diff
@@ -59,6 +59,18 @@ export class Notification extends LitElement {
     error: 'Error',
   };
 
+  /** Close button description */
+  @property({ type: String })
+  closeBtnDescription = '';
+
+  /** Assistive text for timestamp (Required to support accessibility). */
+  @property({ type: String })
+  assistiveTimestampText = '';
```

Comment:
kept this with respect to the following accessibility requirement,
_the duration text should be separately declared as "Duration", just reading the time, someone using screenreader might miss the context_

Reply:
Maybe we should default this to 'Duration' then